lib/numbers.md

```diff
@@ -5,20 +5,44 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+In ruby, there are Fixnum and Float numbers. Fixnum numbers are basically integers, whole numbers, numbers like 5, 23, -9, etc... Float numbers 
+are numbers that contain decimals, such as 3.14, 98.6, 45.5, and so on. 
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+The operations in ruby are very similar to that of regular math operators that most of us are familiar with from school. ("+" for addition, "-" for subtraction, 
+"*" for multiplication, "/" for division, "**" for exponential/"to the power of." More uncommon to non-programmers would be the modulo ("%'), which finds he remainder of given numbers.
+
+Most of the common comparisons should be familiar to most people. For instance: ">" and "<" check to see if the number on the left is greater or lesser than the number on the right, "==" checks for equality while "!=" checks
+for inequality. In addition to this, ">=" and <"= check to see if the number on the left is greater than or equal to (or less than or equality to) the number on the right.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+When using the "+" operation with numbers, the results are what we'd expect: it serves as the addition operation and adds numbers together. However, when using the "+" operation
+with strings, no actual math is performed. Instead, the strings on either side of the "+" operator would be combined, or "concatenated."
+A simple example would be the following:
+"Hello" + "world" would give us a new string: "Hello world"
```

Comment:
Minor nitpick: If you did `"Hello" + "world"`, you would actually get `"Helloworld"` (no space)!

No revision needed–just an FYI.